Yoast/wordpress-seo
# Build a schema tab for the metabox

## Context

* We added a new tab in the metabox that allows users to change their schema settings. This new tab is visible on the following types: posts, pages and custom post types.

## Summary


This PR can be summarized in the following changelog entry:

* Adds a new schema tab to our metabox. This tab allows the user to change their schema settings on posts, pages and custom post types.
* Adds a new section to the search appearance settings. This section allows the user to change their schema defaults for posts, pages and custom post types.

## Relevant technical choices:

*

## Test instructions

TLDR; 
The schema tab should work as expected, this means:
- Saving settings in the metabox should store them in the DB (i.e. they should still be there after reloading the page)
- Those settings should provide the correct output on the frontend (ie. are the correct blocks present?)
- You can change defaults in the search appearance settings. They should also be persisted.
- This default should be mentioned in the individual schema settings.
- If no individual overwrite has been chosen for a post, the default should be present in the output.

For test instructions, check the test instructions for all PRs in the milestone.

This is a list of all PR's in the milestone: https://github.com/Yoast/wordpress-seo/milestone/207?closed=1.

#15737 is not that interesting
#15635 can probably be excluded from testing since it will be noticeable when this is going wrong (schema settings will not save / display correctly).
#15315 can also be skipped since this is included in a complete functionality test

## UI changes

## Documentation

## Quality assurance
